src/components/Thread.js

```diff
@@ -92,7 +92,16 @@ class ThreadInner extends React.PureComponent {
     return null;
   }
 
-  componentDidUpdate(prevProps, prevState, snapshot) {
+  async componentDidUpdate(prevProps, prevState, snapshot) {
+    const parentID = this.props.thread.id;
+    if (
+      parentID &&
+      this.props.thread.reply_count > 0 &&
+      this.props.threadMessages.length === 0
```

Comment:
Just realized that this won't work for the case if someone deletes all the messages in thread!!